packages/whatsie-git/whatsie-git.pacscript

```diff
@@ -0,0 +1,23 @@
+name="whatsie-git"
+gives="whatsie"
+arch=("amd64")
```

Comment:
This is a git package, so it should work on any architecture correct?

Reply:
I believe so, however, I haven't tested it on an arm, so I can't say.

Reply:
It's in C++, so remove the `arch`.